packages/kbn-try-in-console/components/try_in_console_button.tsx

```diff
@@ -84,6 +89,7 @@ export const TryInConsoleButton = ({
   const commonProps = {
     'data-test-subj': type === 'link' ? 'tryInConsoleLink' : 'tryInConsoleButton',
     'aria-label': getAriaLabel(),
+    'data-telemetry-id': telemetryId,
```

Comment:
this is so we have extra information in the click to capture information about the language and code example, when we use the UI-browser-events table.